sir_cake/store/views.py

```diff
@@ -351,6 +354,7 @@ def payment(request):
 
     context['step'] = 3
     context['form'] = payment_form
+    context['expire_date'] = _get_basket_expire_date(preorder)
```

Comment:
If you don't need the preorder anywhere else you can write:
`context['expire_date'] = _get_basket_expire_date(_get_preorder_or_none(request))`